### PR TITLE
Web crawling 3rd PR

### PR DESCRIPTION
**Pull request by JWWJGit**
# 👉 Summary
### Make the investopedia function return 5 links and one txt file containing 5 articles from investopedia.
# 👀 Detail
### Output: 5 links and one txt file containing 5 articles from investopedia.

### CODE REVIEW

File: Phraends_Flask/Backend/Crawler/the_news_texts.txt
Comment (YLee2121) [inline Phraends_Flask/Backend/Crawler/the_news_texts.txt:1]: @tejkoiejgns 

If you agree with this output format for your convenience, simply only leave an approved comment! Then, I will know you've had an agreement with @Weirdojord.

File: Phraends_Flask/Backend/Crawler/Crawler.py
Comment (YLee2121) [inline Phraends_Flask/Backend/Crawler/Crawler.py:1]: @Weirdojord 

I wanted you to write the function comment for `get_news_from_investopedia based on the guideline! Please make sure to write it clearly.

**Review by YLee2121 — CHANGES_REQUESTED** [review]
Request changes

**Comment by JWWJGit** [comment]
Ok no problem!

File: Phraends_Flask/Backend/Crawler/the_news_texts.txt
Comment (tejkoiejgns) [reply Phraends_Flask/Backend/Crawler/the_news_texts.txt:1]: agree!

**Review by YLee2121 — APPROVED** [review]
Approve

**Pull request merged**